File: C++/sums_loop.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
	int arr1[30],arr2[30],sum[30]={0},num;
	cout<<"Enter the number of pairs to be added\n";
	cin>>num;
	for(int i=0;i<num;i++){
		cin>>arr1[i];
		cin>>arr2[i];
		sum[i] = arr1[i]+arr2[i];
	}
	cout<<"The results are : \n ";
	for(int i=0;i<num;i++){
		cout<<sum[i]<<"\t";
	}
}

File: Python/sums_loop.py
num = int(input("Enter how many pairs to be added\n"))
sum = [{0}]
arr1 = []
arr2 = []
for i in range(num):
    arr1.append(int(input()))
    arr2.append(int(input()))
print("The resluts are:\n")
for x,y in zip(arr1,arr2):
    print(x+y,end='\t')
    

   
    
